server/server.go

```diff
@@ -1,57 +1,30 @@
 package server
 
+import (
+	"path/filepath"
+
+	"github.com/mrunalp/ocid/oci"
+)
+
 const (
 	runtimeAPIVersion = "v1alpha1"
 )
 
 // Server implements the RuntimeService and ImageService
 type Server struct {
-	runtime    ociRuntime
-	sandboxDir string
-	sandboxes  []*sandbox
+	runtime   *oci.Runtime
+	sandboxes []*sandbox
```

Comment:
If sandboxDir is a field of `Runtime`, we may need to reconsider the definition of the struct `sandbox`. Its current definition is as follows. The `name` field may only make sense under each sandboxDir.

    type sandbox struct {
        name   string
        logDir string
        labels map[string]string
    }